### ISSUE TITLE
Resolve export members

### ISSUE THREAD

**Issue by jamestalmage** [mattzeunert/OctoTern#5]
This will probably be pretty tricky:

``` js
// my-module.js

module.exports.foo = function () {};
module.exports.bar = function () {};
```

``` js
// index.js
var myModule = require('./my-module.js');

myModule.foo(); // make foo clickable here
```

I'm not sure how best to tackle this, especially since there are all sorts of scenarios where it wouldn't work, and you would have to give up.

I think probably the best is to just linkify anything you think you might be able to resolve, and then make ajax calls for the dependency when they click, try and resolve it, then redirect (popping up a "Sorry!" message when you fail to resolve).

**Comment by mattzeunert** [comment]
I agree this would be nice, but I don't anticipate having time to do this myself.
